### PR TITLE
Feature/plato and search overview fixes

### PR DESCRIPTION
**Pull request by mietcls**
Fixes https://github.com/ugent-library/biblio-backoffice/issues/1588 and feedback on overview page.

Not done yet: adding UGent IDs to authors in candidate suggestions preview

### CODE REVIEW

File: views/publication/summary/summary.templ [hunk 1]
```diff
@@ -132,27 +132,27 @@ templ Summary(c *ctx.Ctx, args SummaryArgs) {
...
+										}
+									>
+										<i class="if if-edit if--small"></i>
+										<em>Add department</em>
+									</a>
+								}
+							</div>
 						</div>
-						<div class="collapsible-card-content collapse" data-collapsible-card-content>
+						if vabb := args.Publication.VABB(); vabb != "" || args.Publication.Legacy || len(args.Publication.RelatedDataset) > 0 {
```
Comment (mietcls) [inline views/publication/summary/summary.templ:154]: Not sure if this is the best solution to hide the block when nothing is in there.

File: views/publication/preview.templ [hunk 2]
```diff
@@ -350,28 +350,30 @@ templ Preview(c *ctx.Ctx, p *models.Publication, actions, downloadMainFileAction
...
-																				</li>
-																			}
-																			if a.ORCID() != "" {
-																				<li class="c-meta-item text-nowrap">
-																					<i class="if if-orcid if--small if--success"></i>
-																					<span>{ a.ORCID() }</span>
-																				</li>
-																			}
-																		</ul>
+																		if a.Person != nil || a.ORCID() != "" {
```
Comment (mietcls) [inline views/publication/preview.templ:361]: Not sure if this is the best solution.

File: views/publication/preview.templ [hunk 3]
```diff
@@ -363,7 +363,7 @@ templ Preview(c *ctx.Ctx, p *models.Publication, actions, downloadMainFileAction
 																				if a.Person != nil {
 																					<li class="c-meta-item">
 																						<i class="if if-ghent-university if--small if--muted"></i>
-																						<span class="text-muted">UGent author</span>
+																						<span class="text-muted">UGent</span>
```
Comment (mietcls) [inline views/publication/preview.templ:366]: Cannot test this right now as none of the suggestions have users with an UGent ID.

File: views/publication/preview.templ (hunk 2 above)
Comment (nics) [reply views/publication/preview.templ:361]: it's fine

File: views/publication/summary/summary.templ (hunk 1 above)
Comment (nics) [reply views/publication/summary/summary.templ:154]: this won't work correctly
the conditions ` || args.Publication.Legacy || len(args.Publication.RelatedDataset) > 0` will only be evaluated when `vabb != ""`

Comment (nics) [reply views/publication/summary/summary.templ:154]: better add a helper function like `func showX(p *models.Publication) bool`

**Review by nics — APPROVED** [review]

**Pull request merged**